docs/features/type-ignore-enhancements.md

```
@@ -0,0 +1,303 @@
+## 概要
+
+現在の`pylay check --focus ignore`機能を拡張し、単なる検出だけでなく、具体的なリファクタリング提案を行う機能を追加します。
+
+## 現在の機能
+
+```bash
+uv run pylay check --focus ignore
+```
+
+**出力例**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)